.github/workflows/deploy-polaris.shopify.com.yml

```diff
@@ -21,5 +21,9 @@ jobs:
               owner: "shopify",
               repo: "polaris-site-container-builder",
               workflow_id: "trigger-deploy.yml",
-              ref: "main"
+              ref: "main",
+              inputs: {
+                sha: context.sha,
+                target: 'production'
+              }
```

Comment:
These inputs will be consumed by shopify/polaris-site-container-builder once we have merged changes there. For now, they'll simply be ignored.